asteroid/utils/hub_utils.py

```diff
@@ -71,8 +62,10 @@ def cached_download(filename_or_url):
         else:
             model_id = filename_or_url
             revision = None
-        url = hf_bucket_url(model_id=model_id, filename=HF_WEIGHTS_NAME, revision=revision)
-        return hf_get_from_cache(url, cache_dir=get_cache_dir())
+        url = huggingface_hub.hf_hub_url(
+            model_id, filename=huggingface_hub.file_download.PYTORCH_WEIGHTS_NAME, revision=revision
```

Comment:
I can expose `PYTORCH_WEIGHTS_NAME` in the library's init.py so that you don't have to import from a nested file

Reply:
That would be great.